.rubocop.yml

```diff
@@ -0,0 +1,6 @@
+StringLiterals:
+  Enabled: false
+
+LineLength:
```

Comment:
Also, I'm fine with leaving this switched off for the time being, but I think we should aim at re-enabling this check at a later time (in the not too distant future) and make sure we stay within an 80-character boundary.